### PR TITLE
[UX] Gesture manager: add action - fulltext search

### LINKED ISSUE #4687: Even more Gesture Manager action suggestions
Now that the Gesture Manager has been massively revamped, here are some more features I personally use often:
- [x] Wikipedia lookup
- [x] Dictionary lookup
- [x] Toggle Reflow

There are more, but the action list is already pretty long at this point. Probably shouldn't bloat it with niche features.

### PR DESCRIPTION
**Pull request by Frenzie**
Fixes #4687.

### CODE REVIEW

File: frontend/apps/reader/modules/readersearch.lua [hunk 1]
```diff
@@ -21,14 +21,20 @@ end
...
-            end,
-        },
+        callback = function()
+            self:onShowSearchInput()
+        end,
+    }
+end
+
+function ReaderSearch:onShowFulltextSearchInput()
+    self:onInput{
```
Comment (Frenzie) [inline frontend/apps/reader/modules/readersearch.lua]: @poire-z This definitely looks tidier. Maybe dictionary/Wikipedia should also do it this way. :-)

Comment (poire-z) [reply frontend/apps/reader/modules/readersearch.lua]: Feel free :)

File: frontend/apps/reader/modules/readersearch.lua [hunk 2]
```diff
@@ -21,14 +21,20 @@ end
...
-        tap_input = {
-            title = _("Input text to search for"),
-            ok_text = _("Search all text"),
-            type = "text",
-            callback = function(input)
-                self:onShowSearchDialog(input)
-            end,
-        },
+        callback = function()
+            self:onShowSearchInput()
```
Comment (poire-z) [inline frontend/apps/reader/modules/readersearch.lua]: `readersearch.lua:25: attempt to call method 'onShowSearchInput' (a nil value)`

Comment (Frenzie) [reply frontend/apps/reader/modules/readersearch.lua]: My bad, I changed the name last second because I thought it sounded too generic this way.

**Pull request merged**